### PR TITLE
Update dependency on cardano-ledger-specs

### PR DESCRIPTION
**Pull request by mrBliss**

### CODE REVIEW

**Comment by mrBliss** [comment]
Draft because the `EraGen` instance for Allegra and Mary are missing from the ledger. We'll have to wait for https://github.com/input-output-hk/cardano-ledger-specs/pull/2012.

**Comment by mrBliss** [comment]
Rebased against the latest master, updated the index-state. Still waiting on https://github.com/input-output-hk/cardano-ledger-specs/pull/2012.

File: cabal.project [hunk 1]
```diff
@@ -152,8 +152,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger-specs
-  tag: 9ebcfedb3f93852fbeabfc8ce492ddb4d996b3b5
-  --sha256: 03n1m2030liv55b2jjjc7ga8w3fkbcxr6xpgsxa0gb9lhx41r4m5
+  tag: 95b6b070459da13824236b5e50dffebfeee080da
+  --sha256: 18c644ag2i7i1cddh88pbqsm9gakpg686lrxjwh8s30b7kmkb3nd
   subdir:
```
Comment (mrBliss) [inline cabal.project:157]: Update the other dependencies too?

**Review by edsko — APPROVED** [review]

**Comment by mrBliss** [comment]
Waiting on https://github.com/input-output-hk/cardano-ledger-specs/pull/2029

**Comment by mrBliss** [comment]
Rebased on master. Still waiting for https://github.com/input-output-hk/cardano-ledger-specs/pull/2029 to be merged.

**Comment by mrBliss** [comment]
bors merge

**Pull request merged**